recipes/GeoTessCPP/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+git config --global --add safe.directory /home/conda/staged-recipes
```

Comment:
Is this really needed?

Reply:
Thought I needed it for local Linux build to finish.  Removing.